From e0fc8008157d4393b24647c17dfd2db98217e03f Mon Sep 17 00:00:00 2001
From: Schuyler Patton <spatton@ti.com>
Date: Thu, 30 Jun 2011 14:55:54 -0500
Subject: [PATCH 2/3] omap3_evm: Changed default clock rate for AM3517

* Changed #define MPU_M_13_ES2 from 0x1F4 to 0x258, this allows
  the AM3517 to boot up at 600MHz instead of 500 MHz

Upstream-Status: Pending

Signed-off-by: Schuyler Patton <spatton@ti.com>
---
 arch/arm/include/asm/arch-omap3/clocks_omap3.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/include/asm/arch-omap3/clocks_omap3.h b/arch/arm/include/asm/arch-omap3/clocks_omap3.h
index e9784f0..4073416 100644
--- a/arch/arm/include/asm/arch-omap3/clocks_omap3.h
+++ b/arch/arm/include/asm/arch-omap3/clocks_omap3.h
@@ -74,7 +74,7 @@
 #define MPU_FSEL_13_ES1		0x03
 #define MPU_M2_13_ES1		0x01

-#define MPU_M_13_ES2		0x1F4
+#define MPU_M_13_ES2		0x258
 #define MPU_N_13_ES2		0x0C
 #define MPU_FSEL_13_ES2		0x03
 #define MPU_M2_13_ES2		0x01
--
1.7.0.4
